docs/guides/tools/proposer-based-timestamps-runbook.md

```diff
@@ -106,7 +106,15 @@ Re-run the `timedatectl` command and verify that the change has taken effect.
 
 #### Check if Your NTP Daemon is Synchronized
 
-Check the status of your local `chrony` NTP daemon using by running the following:
+We provide two examples here, for `chrony` and `timesync`, but these steps
+should be adapted if you are using a different daemon.
+
+If you find that the NTP is not synchronizing, remember allow NTP traffic
```

Comment:
```suggestion
If you find that the NTP is not synchronizing, remember allow NTP inbound traffic
```

Reply:
Not sure how to phrase this. The traffic is initiated from inside, so I should say outbound traffic. But we need to allow the returning traffic as well, it cannot be just outbound. Cannot even say "allow connecting to NTP servers" since it uses UDP. I think I will just leave it this way.

Reply:
Yup, but since that's the nature of NTP, I will just leave it like this and let net admins decide how they implement it.